src/handlers/issue-deduplication.ts

```diff
@@ -145,6 +226,7 @@ async function handleSimilarIssuesComment(
 
   if (relevantIssues.length === 0) {
     context.logger.info("No relevant issues found with the same repository and organization");
+    return;
```

Comment:
I originally designed the logger to be used like this

throw context.logger.info("...